### PR TITLE
Fix #3164: Restore my CODEOWNERS

### LINKED ISSUE #3164: Restore code ownership after 24 May 2021
I will be working at reduced hours & taking time off over the next couple weeks, and should be able to resume after 24 May.

### PR DESCRIPTION
**Pull request by BenHenning**
Fix #3164

## Explanation
Restoring my codeowners per #3164 now that GSoC M1 is about to wrap up. Note that this also refines script ownership since there have been a lot of changes in that directory. @anandwana001 now co-owns the shell files, but everything else (the new items) will be under my ownership for now.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by BenHenning** [comment]
/cc @rt4914 @anandwana001 @vinitamurthi 

I'll be putting this in review once GSoC M1 PRs are merged to avoid churn.

**Comment by BenHenning** [comment]
@seanlip PTAL when you get a chance. The only 2 M1 PRs left in flight either have or will soon have my approval, so I think we can make this change now.

**Review by seanlip — APPROVED** [review]

**Pull request merged**
